libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Extension methods for QnA.
+    /// </summary>
+    public static class QnAMakerExtensions
+    {
+        /// <summary>
+        /// Setup configuration to utilize the settings file generated by bf qnamaker:build.
+        /// </summary>
+        /// <remarks>
+        /// This will pick up --environment as the environment to target.  If environment is Development it will use the name of the logged in user.
+        /// This will pick up --root as the root folder to run in.  
+        /// </remarks>
+        /// <param name="builder">Configuration builder to modify.</param>
+        /// <param name="botRoot">bot Root.</param>
+        /// <param name="qnaRegion">qnaRegion.</param>
+        /// <param name="environment">enviroment.</param>
+        /// <returns>Modified configuration builder.</returns>
+        public static IConfigurationBuilder UseQnAMakerSettings(this IConfigurationBuilder builder, string botRoot, string qnaRegion, string environment)
```

Comment:
I don't have a lot of context on this approach, but if this is trying to be similar to LuisExtensions, why doesn't it have the same arguments? shouldn't the look and work the same?

Reply:
We do not want to revise the existing luis extension. But we hope to set the default values for qna maker extension  in testbot.json.